packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -113,9 +126,10 @@ TODO:
         return () => resizer?.disconnect();
     }
 
-    function sourceLoadingCallback(e) {
+    function sourceLoadingCallback(e: MapSourceDataEvent) {
         // sourceDataType can be "visibility" or "metadata", in which case it's not about the data itself
         if (e.isSourceLoaded && e.sourceId === sourceId && !e.sourceDataType) {
+            // @ts-ignore // The type forces you to pass a filter parameter in the option, but it's not required by the real code
```

Comment:
There's an open issue on this subject: https://github.com/maplibre/maplibre-gl-js/issues/1393

Could we add a comment pointing to this issue in order to remove the `@ts-ignore` once they fix it?

Reply:
Thanks, didn't see that one!